src/RunScriptCommand.cs

```diff
@@ -77,32 +79,29 @@ public async Task<int> InvokeAsync(InvocationContext context)
             return 0;
         }
 
-        // The `env` script is special so if it's not explicitly declared we act like it was
-        var scriptsToRun = scripts
-            .Select(script => (name: script, exists: project.Scripts!.ContainsKey(script) || script == "env"))
```

Comment:
Turns out this (`script == "env"`) is a bug and was fixed below in the new version. Long term the case insensitive support should probably be removed to both simplify things and reduce the number of edge case bugs like this.